internal/testutils/metric_testutils.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
```

Comment:
Consider package "metricstestutils"

Reply:
Since testutils.go has package testutils, I need to use the same package name.

Reply:
Or did you mean a new directory/package i.e. - internal/metricstestutils/

Reply:
Yeah, having this in a separate package will be better. This package also seems useful for anyone writing any metrics component. We should consider moving it to a location out of `internal` but that can come later.

Reply:
Done.